website/blog/2023-06-30-data-vault-2-0-with-dbt-cloud.md

```diff
@@ -0,0 +1,146 @@
+
+## How to get started with dbt Cloud and Data Vault
+
+There are many decisions to make before you roll up your sleeves and start implementing your Data Vault data warehouse. Apart from data modelling work, you need to agree on naming conventions, hash algorithm, staging strategy and data types for standard metadata attributes, and make sure these are all well documented. Here, to save yourself some headaches in the long run, we recommend starting your own **decision log**.
+
+In terms of the implementation of the Data Vault itself, we recommend familiarising yourself with the best practices well in advance, especially if you have no previous experience with the framework. There are two well known dbt packages focusing on Data Vault implementation, which you can take inspiration from to build your own templating system, or there can be used directly if they fit your use case.
+
+### AutomateDV (formerly known as dbtvault)
+
+AutomateDV is the most popular open source Data Vault package for dbt, with some users having over 5000 Data Vault components in their project. Here in Infinite Lambda, we’ve been using this package for quite some time now, even building on top of it (depending on the specifics of the project). This mature system provides a great way to start your Data Vault with dbt Cloud journey as the learning curve is quite manageable, it is well documented and even comes with tutorial and working examples built on top of Snowflake’s TPCH standard dataset. There is one limitation to using the package and that is _AutomateDV _expects your source data to contain only one delta load. In order to workaround this issue, owners of the package came up with custom dbt materialisations helping you with initial load of your system, however the performance of such load is in our experience not acceptable. 
+
+### datavault4dbt
+
+At first glance, this fairly new open source package works in a similar fashion, especially the usage of the macros provides the same experience (apart from names of some of the parameters). Diving deeper into documentation however, we can see it provides a higher level of customisation thanks to many global variables, which alters the behaviour of macros. It also supports any type of source data - CDC, transient or persistent, it can handle it all. We suggest looking into this package if you have a deeper understanding of Data Vault and need a complex, customisable system. It’s good to be aware of the fact that this package is new, so there is a risk of hidden unresolved issues. 
```

Comment:
```suggestion
At first glance, this fairly new open source package works in a similar fashion, especially since the usage of the macros provides the same experience (apart from the names of some of the parameters). Diving deeper into documentation, however, we can see it provides a higher level of customization thanks to many global variables, which alters the behavior of macros. It also supports any type of source data - CDC, transient or persistent, it can handle it all. We suggest looking into this package if you have a deeper understanding of Data Vault and need a complex, customizable system. It’s good to be aware of the fact that this package is new, so there is a risk of hidden unresolved issues. 
```